Week-5/asyncawait.js

```diff
@@ -0,0 +1,42 @@
+function doTask1(num) {
```

Comment:
Can you create two sets of doTask functions in two different files, one for generators and async await.

And for async await, make the function as async.

Reply:
But it looks good to me. Just the way its written